docs/synchronization.md

```diff
@@ -147,8 +150,64 @@ Examples:
 1. [Step level semaphore](https://github.com/argoproj/argo-workflows/blob/main/examples/synchronization-tmpl-level.yaml)
 1. [Step level mutex](https://github.com/argoproj/argo-workflows/blob/main/examples/synchronization-mutex-tmpl-level.yaml)
 
-### Other Parallelism support
+### Multiple synchronization
```

Comment:
This needs a version specifier

Reply:
I've moved this down from however it ended up in the top of the file